$NetBSD$

Don't override docdir when prefix is set
Fixed the lack of build and host options.
It always failed to find patch even though it's there.

--- configure.orig	2023-12-22 16:18:46.000000000 +0000
+++ configure
@@ -682,8 +682,7 @@ while : ; do
       zipdir=$libdir/$fildir;
       dlldir=$libdir/$fildir;
       mandir=$prefix/man/man1;
-      infodir=$prefix/info;
-      docdir=$prefix/doc/bigloo-$release;;
+      infodir=$prefix/info;;
 
     --bigloo=*)
       bigloo="`echo $1 | sed 's/^[^=]*=//'`";;
@@ -1120,6 +1119,9 @@ while : ; do
     --enable-srfi27)
       srfi27force="yes";;
 
+    --enable-option-checking=*)
+      ;;
+
     --enable-*)
       api="`echo $1 | sed 's/--[^-]*-//'`";
       add=1;
@@ -1151,6 +1153,12 @@ while : ; do
     --hostsh=*)
       hostsh="`echo $1 | sed 's/^[^=]*=//'`";;
 
+    --build=*)
+      ;;
+
+    --host=*)
+      ;;
+
     -*)
       if [ "$1" != "-h" -a "$1" != "-help" -a "$1" != "--help" ]; then
         echo "*** ERROR: unknown option \"$1\""
@@ -2119,18 +2127,18 @@ if [ $action = "all" -o $action = "biglo
 fi
 
 # checking patch
-patch=$PATCH
-
-if [ "$patch " = " " ]; then
-  patch=patch
-fi
+#patch=$PATCH
 
-$patch --help 2> /dev/null > /dev/null
-
-if [ $? != 0 ]; then
-  $echo "*** ERROR: patch ($patch) command not found. Aborting..." >&2;
-  exit 1;
-fi
+#if [ "$patch " = " " ]; then
+#  patch=patch
+#fi
+
+#$patch --help 2> /dev/null > /dev/null
+
+#if [ $? != 0 ]; then
+#  $echo "*** ERROR: patch ($patch) command not found. Aborting..." >&2;
+#  exit 1;
+#fi
   
 #*---------------------------------------------------------------------*/
 #*    Executable names                                                 */
@@ -2242,7 +2250,7 @@ else
   fi
 
   if [ "$gcparallelmark " = "yes " ]; then
-    gcthread="$gcthread -DPARALLEL_MARK";
+    gcthread="-DPARALLEL_MARK";
   fi
 
   # thread local storage
